portal-backend/depmap/breadbox_shim/breadbox_shim.py

```diff
@@ -34,15 +35,21 @@ class BreadboxVectorCatalogChildNode(Node):
     for breadbox nodes, so a simpler version is being used as a return type in the shim.
     """
 
-    def __init__(self, slice_id: str, label: str, is_terminal: bool):
+    def __init__(
+        self,
+        slice_id: str,
+        label: str,
+        value: str,  # For datasets, this should be the ID, for features, it's the label
+        is_terminal: bool,
+    ):
         # Fill in defaults for values that are required in the Node type
         super().__init__(
             id=slice_id,
             key=None,
             attrs={},
             is_terminal=is_terminal,
             label=label,
-            value=label,
+            value=value,  # maps onto childValue in the API response
```

Comment:
This change fixes an issue with the "View Results in DE2" button, which reads from this `"childValue"` field.